M2/Macaulay2/m2/undocs.m2

```diff
@@ -0,0 +1,327 @@
+restart
```

Comment:
Was this file intended to be commited to git?

Reply:
Certainly not haha